daemons/validator_score_mainnet_v2_daemon.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require_relative "../config/environment"
+include ValidatorScoreV2Logic
+include PipelineLogic
+
+# Send an interrupt with `ctrl-c` or `kill` to stop the script. Results will
+# not be posted to the validators.app server.
+interrupted = false
+trap("INT") { interrupted = true }  unless Rails.env.test?
+
+network = "mainnet"
+sleep_time = 15 # seconds
+
+class SkipAndSleep < StandardError; end
+
+begin
+  loop do
```

Comment:
I think we should move code from the loop to the service and include it in daemons with correct params (network and queue in worker is the only difference I think) to avoid maintaining the same code in two places.

Reply:
I wrote it this way ob purpose. I think it'll be easier to decide which code to delete when we'll move to v2, if we keep them as separate as possible. Eg. in this case some files will be just to delete with one click without thinking which code to delete and which to keep.

Reply:
Sounds good, but please remember to sort it out after transition.